languages/common-lisp/exercises/concept/sameness/.docs/hints.md

```diff
@@ -0,0 +1,19 @@
+## What is one supposed to do again?
```

Comment:
A fantastic hints file! I really love the headers and the general tone!

With that being said, I was told for the basics exercise to line up the hints with each of the parts (by putting them under the same named header as in the instructions.md) [This is what Erik ended up approving](https://github.com/exercism/v3/blob/master/languages/common-lisp/exercises/concept/basics/.docs/hints.md)

Maybe we could have the best of both worlds by having the headers you currently have as sub-headers of the `introduction.md` sections?

Reply:
Hmm there aren't really sections to this exercise. you just do it. If anything there might be a section for each test.

Or we could rearrange the exercise to have sections, one for each generic predicate, or one for each type we talk about (characters, arrays). Then the hints could reference them.

I'll need to think about it some more.